### ISSUE TITLE
In examples/models/bayesian_vgg.py How does one compute the loss of the model?

### ISSUE THREAD

**Issue by BelhalK** [tensorflow/probability#385]
Hello,

Once defined as
```
x = tf.keras.layers.Flatten()(x)
  x = tfp.layers.DenseFlipout(
      num_classes,
      kernel_posterior_fn=kernel_posterior_fn)(x)
  model = tf.keras.Model(inputs=image, outputs=x, name='vgg16')
```
How do you compute the elbo Loss?
Does `model.losses` work?
which then would lead to this computation
```
labels_distribution = tfd.Categorical(logits=logits)
    neg_log_likelihood = -tf.reduce_mean(input_tensor=labels_distribution.log_prob(abels))
kl = sum(model.losses) / mnist_data.train.num_examples
elbo_loss = neg_log_likelihood + kl
self._loss_op = elbo_loss
```
